ext/node/polyfills/_fs/_fs_close.ts

```diff
@@ -11,7 +11,7 @@ export function close(fd: number, callback: CallbackWithError) {
   setTimeout(() => {
     let error = null;
     try {
-      Deno.close(fd);
+      Deno[Deno.internal].core.close(fd);
```

Comment:
```ts
import { core } from "ext:core/mod.js";

// ...

// TODO: Treat `fd` as real file descriptor. `rid` is an implementation detail and may change.
core.close(fd);
```

Reply:
`ensureFastOps()`?

Reply:
`core.close` should work fine, it internally calls an op returned by `ensureFastOps()`